src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -90,7 +199,15 @@
       <div class="ecl-banner__content">
         {% if _title is not empty %}
           <div class="ecl-banner__title">
-            <span class="ecl-banner__title-text">{{ _title }}</span>
+            <span class="ecl-banner__title-text">
+            {%- if _title.link is defined -%}
+              {% include '@ecl/link/link.html.twig' with _title|merge({
+                extra_classes: 'ecl-banner__title-link'
```

Comment:
this is a link object, right, so it might already contain extra_classes, we should merge them with the one we add here.